docs/source-2.0/guides/converting-to-openapi.rst

```diff
@@ -1176,6 +1176,33 @@ dependency on ``software.amazon.smithy:smithy-aws-apigateway-openapi``.
 Amazon API Gateway configuration settings
 =========================================
 
+apiGatewayDefault (``string``)
+    Sets recommended default configuration settings and allows for those defaults
+    to be disabled. If this setting is omitted, the ``2023-08-11`` version will
+    be used by default.
+
+    The ``2023-08-11`` version sets the following configuration settings:
+
+    * :ref:`alphanumericOnlyRefs <generate-openapi-jsonschema-setting-alphanumericOnlyRefs>`: ``true``
+    * :ref:`disableDefaultValues <generate-openapi-setting-disableDefaultValues>`: ``true``
+    * :ref:`disableIntegerFormat <generate-openapi-setting-disableIntegerFormat>`: ``true``
+    * :ref:`disableFeatures <generate-openapi-jsonschema-setting-disableFeatures>`: ``["default"]``
+
+    .. code-block:: json
+        :caption: smithy-build.json
+
+        {
+            "version": "1.0",
+            "plugins": {
+                "openapi": {
+                    "service": "example.weather#Weather",
+                    "apiGatewayDefault": "2023-08-11"
+                }
+            }
+        }
+
+    If the version is set to ``DISABLED``, the configuration settings will not be set.
```

Comment:
Would be good to make this a list and document "disabled" just like "2023-08-11". Also, is it "DISABLED" or "disabled"? The toString method is "disabled". Might be good to also add an ".. important" admonition to call out setting this explicitly.

Reply:
Made this `DISABLED` consistency with the other settings. `apiGatewayType` uses REST|HTTP|DISABLED.